spec/latest/core/index.html

```diff
@@ -1039,6 +1039,44 @@ <h3>Templated Links</h3>
       }
       -->
     </pre>
+
+    <p>Iri expansion should be performed with respect to the specification
+      behind the iri template type (which is RFC 6570 by default), and the
+      product of this process SHOULD be an Iri. In may happen that this
+      product is a relative Uri. In such a case client by default SHOULD
+      stick to RFC 3986. It is due to fact that most RDF serialization
+      allowing other that absolute Iris will behave that way and the result
+      of Iri template expansion should be compliant. Still, it may be
+      preferred to use other base Uri for the expansion process, which
+      makes a <i>resolveRelativeTo</i> term useful. It allows to switch the
+      Iri template expansion algorithm so the base Uri is established using
+      current link context, which is a subject of the relation pointing to
+      <i>IriTemplate</i> instance. In case that subject is a relative Uri,
+      default behavior SHOULD be used as fallback.</p>
```

Comment:
I took the liberty to reword the paragraph a bit and made `IRI` and `URI` abbreviations consistenly all-uppercase.

```suggestion
    <p>IRI expansion should be performed with respect to the specification
      behind the IRI template type (RFC 6570 by default), and the product of
      this process SHOULD be an IRI. When the produced IRI is relative, a case
      client SHOULD stick to RFC 3986 to be compatible with most RDF
      serializations that support relative IRIs. Still, it may be preferred to
      use another base URI for the expansion process, which makes a
      <i>resolveRelativeTo</i> term useful. It allows to switch the IRI
      template expansion algorithm so the base URI is established using the
      current link context, which is a subject of the relation pointing to an
      <i>IriTemplate</i> instance. In case that the subject is a relative URI,
      the default behavior SHOULD be used as fallback.
    </p>
```